src/openms/include/OpenMS/KERNEL/MSExperiment.h

```diff
@@ -545,6 +545,13 @@ namespace OpenMS
     */
     void clear(bool clear_meta_data);
 
+    /// returns true if at least one of the spectra has level==1
+    bool containsMS1Scans() const;
```

Comment:
I would be fine with having this in MSExperiment. But maybe more generic by passing the level?